src/Integration/Binding/BindingProcessImpl.cs

```diff
@@ -196,7 +216,7 @@ public bool FinishSolutionBindingOnUIThread()
         #region Private methods
 
         internal /* for testing */ IEnumerable<Language> GetBindingLanguages()
-            => host.SupportedPluginLanguages;
```

Comment:
I changed this code to return this value earlier in the sprint (previously there was a lot of complicated code to walk the projects in the solution and find the languages that are actually used).

However, I now realise my original change is original. I should have changed it to return the list of all languages known to SLVS.

This change fixes that.